Tests/Marketplace/Tests/search_and_unistall_pack_test.py

```diff
@@ -112,7 +112,7 @@ def test_exception_reset_base_pack(def_call, return_val, mocker):
    """
     client = MockClient()
     client.api_client.configuration.host = 'https://api-someurl.com/'  # disable-secrets-detection
-    ret_val_from_call = def_call(client)
+    ret_val_from_call = def_call(client, ['Base'])
```

Comment:
more the `['Base']` list to be a parameter for the test, and test for more scenarios: Empty, 1 pack, 2 packs.